### ISSUE TITLE
rubberband: "ladspa.h: No such file or directory"

### ISSUE THREAD

**Issue by slhck** [Linuxbrew/homebrew-core#11609]
- [x] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Linuxbrew/homebrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link? --> https://gist.github.com/slhck/00679117f06ad1fa58535afd7c6b62e1
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:

- What you were trying to do (and why): `brew install rubberband`
- What happened (include command output):

```
➜ brew install rubberband 
==> Downloading https://breakfastquay.com/files/releases/rubberband-1.8.2.tar.bz2
Already downloaded: /home/werner/.cache/Homebrew/downloads/d5120ec06f35130415c35ee046068847e2a4c5f172b470f36971e0270fb8348c--rubberband-1.8.2.tar.bz2
==> ./configure --disable-silent-rules --prefix=/home/linuxbrew/.linuxbrew/Cellar/rubberband/1.8.2_1
==> make install
Last 15 lines from /home/werner/.cache/Homebrew/Logs/rubberband/02.make:
g++-8 -DHAVE_LIBSAMPLERATE -DHAVE_FFTW3 -DFFTW_DOUBLE_ONLY -DNO_THREAD_CHECKS -DUSE_PTHREADS -DNO_TIMING -DNDEBUG -I. -Isrc -Irubberband -g -O2 -fPIC -Wall -I/home/linuxbrew/.linuxbrew/Cellar/libsamplerate/0.1.9_1/include -I/home/linuxbrew/.linuxbrew/Cellar/libsndfile/1.0.28/include -I/home/linuxbrew/.linuxbrew/Cellar/fftw/3.3.8/include -I/home/linuxbrew/.linuxbrew/Cellar/vamp-plugin-sdk/2.7.1/include    -c -o src/system/sysutils.o src/system/sysutils.cpp
g++-8 -DHAVE_LIBSAMPLERATE -DHAVE_FFTW3 -DFFTW_DOUBLE_ONLY -DNO_THREAD_CHECKS -DUSE_PTHREADS -DNO_TIMING -DNDEBUG -I. -Isrc -Irubberband -g -O2 -fPIC -Wall -I/home/linuxbrew/.linuxbrew/Cellar/libsamplerate/0.1.9_1/include -I/home/linuxbrew/.linuxbrew/Cellar/libsndfile/1.0.28/include -I/home/linuxbrew/.linuxbrew/Cellar/fftw/3.3.8/include -I/home/linuxbrew/.linuxbrew/Cellar/vamp-plugin-sdk/2.7.1/include    -c -o src/system/Thread.o src/system/Thread.cpp
g++-8 -DHAVE_LIBSAMPLERATE -DHAVE_FFTW3 -DFFTW_DOUBLE_ONLY -DNO_THREAD_CHECKS -DUSE_PTHREADS -DNO_TIMING -DNDEBUG -I. -Isrc -Irubberband -g -O2 -fPIC -Wall -I/home/linuxbrew/.linuxbrew/Cellar/libsamplerate/0.1.9_1/include -I/home/linuxbrew/.linuxbrew/Cellar/libsndfile/1.0.28/include -I/home/linuxbrew/.linuxbrew/Cellar/fftw/3.3.8/include -I/home/linuxbrew/.linuxbrew/Cellar/vamp-plugin-sdk/2.7.1/include    -c -o src/StretcherChannelData.o src/StretcherChannelData.cpp
g++-8 -DHAVE_LIBSAMPLERATE -DHAVE_FFTW3 -DFFTW_DOUBLE_ONLY -DNO_THREAD_CHECKS -DUSE_PTHREADS -DNO_TIMING -DNDEBUG -I. -Isrc -Irubberband -g -O2 -fPIC -Wall -I/home/linuxbrew/.linuxbrew/Cellar/libsamplerate/0.1.9_1/include -I/home/linuxbrew/.linuxbrew/Cellar/libsndfile/1.0.28/include -I/home/linuxbrew/.linuxbrew/Cellar/fftw/3.3.8/include -I/home/linuxbrew/.linuxbrew/Cellar/vamp-plugin-sdk/2.7.1/include    -c -o src/StretcherImpl.o src/StretcherImpl.cpp
g++-8 -DHAVE_LIBSAMPLERATE -DHAVE_FFTW3 -DFFTW_DOUBLE_ONLY -DNO_THREAD_CHECKS -DUSE_PTHREADS -DNO_TIMING -DNDEBUG -I. -Isrc -Irubberband -g -O2 -fPIC -Wall -I/home/linuxbrew/.linuxbrew/Cellar/libsamplerate/0.1.9_1/include -I/home/linuxbrew/.linuxbrew/Cellar/libsndfile/1.0.28/include -I/home/linuxbrew/.linuxbrew/Cellar/fftw/3.3.8/include -I/home/linuxbrew/.linuxbrew/Cellar/vamp-plugin-sdk/2.7.1/include    -c -o main/main.o main/main.cpp
g++-8 -DHAVE_LIBSAMPLERATE -DHAVE_FFTW3 -DFFTW_DOUBLE_ONLY -DNO_THREAD_CHECKS -DUSE_PTHREADS -DNO_TIMING -DNDEBUG -I. -Isrc -Irubberband -g -O2 -fPIC -Wall -I/home/linuxbrew/.linuxbrew/Cellar/libsamplerate/0.1.9_1/include -I/home/linuxbrew/.linuxbrew/Cellar/libsndfile/1.0.28/include -I/home/linuxbrew/.linuxbrew/Cellar/fftw/3.3.8/include -I/home/linuxbrew/.linuxbrew/Cellar/vamp-plugin-sdk/2.7.1/include    -c -o vamp/RubberBandVampPlugin.o vamp/RubberBandVampPlugin.cpp
g++-8 -DHAVE_LIBSAMPLERATE -DHAVE_FFTW3 -DFFTW_DOUBLE_ONLY -DNO_THREAD_CHECKS -DUSE_PTHREADS -DNO_TIMING -DNDEBUG -I. -Isrc -Irubberband -g -O2 -fPIC -Wall -I/home/linuxbrew/.linuxbrew/Cellar/libsamplerate/0.1.9_1/include -I/home/linuxbrew/.linuxbrew/Cellar/libsndfile/1.0.28/include -I/home/linuxbrew/.linuxbrew/Cellar/fftw/3.3.8/include -I/home/linuxbrew/.linuxbrew/Cellar/vamp-plugin-sdk/2.7.1/include    -c -o vamp/libmain.o vamp/libmain.cpp
g++-8 -DHAVE_LIBSAMPLERATE -DHAVE_FFTW3 -DFFTW_DOUBLE_ONLY -DNO_THREAD_CHECKS -DUSE_PTHREADS -DNO_TIMING -DNDEBUG -I. -Isrc -Irubberband -g -O2 -fPIC -Wall -I/home/linuxbrew/.linuxbrew/Cellar/libsamplerate/0.1.9_1/include -I/home/linuxbrew/.linuxbrew/Cellar/libsndfile/1.0.28/include -I/home/linuxbrew/.linuxbrew/Cellar/fftw/3.3.8/include -I/home/linuxbrew/.linuxbrew/Cellar/vamp-plugin-sdk/2.7.1/include    -c -o ladspa/RubberBandPitchShifter.o ladspa/RubberBandPitchShifter.cpp
In file included from ladspa/RubberBandPitchShifter.cpp:24:
ladspa/RubberBandPitchShifter.h:27:10: fatal error: ladspa.h: No such file or directory
 #include <ladspa.h>
          ^~~~~~~~~~
compilation terminated.
make: *** [<builtin>: ladspa/RubberBandPitchShifter.o] Error 1
make: *** Waiting for unfinished jobs....
```

- What you expected to happen: Installation succeeds
- Step-by-step reproduction instructions (by running `brew install` commands): `brew install rubberband`

**Comment by JuPlutonic** [comment]
1. Same issue here.
Maybe you could overcome it with carla or lrdf packages from linuxbrew.
```CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix $FORMULA_NAME)" CFLAGS='-g -O2' RUBY_CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix $FORMULA_NAME)" brew install rubberband```
Where ```FORMULA_NAME=carla```: Audio plugin host supporting LADSPA, LV2, VST2/3, SF2 and more. Or ```lrdf``` : RDF library for accessing plugin metadata in the LADSPA plugin system.

2. Now on Linuxbrew [2.0](https://brew.sh/2019/02/02/homebrew-2.0.0/) some updates come in unstable shape. And we wait for bottles from maintainers.

3. Maybe valuable for you will be line ```export HOMEBREW_INSTALL_CLEANUP=0``` in ```~/.profile``` file and when you like to upgrade linuxbrew after you found out what after upgrade no errors happend and all formulae are in place(in Cellar dir) when may remove remaining old formulae versions-directories with ```brew cleanup``` command.

**Comment by JuPlutonic** [comment]
Tried brew install lrdf; CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix lrdf)" CFLAGS='-g -O2' RUBY_CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix lrdf)" brew install rubberband
gist-logs()

**Comment by iMichka** [comment]
I already made an attempt to get this to work #7923. It looks like we need to revive my attempt to add the ladspa formula.

**Comment by iMichka** [comment]
I got ladspa to build: #11612

I will pull the bottle as soon as possible and start working on rubberband.

**Comment by slhck** [comment]
That's awesome, thanks for your help! I wouldn't even know where to start.

**Comment by iMichka** [comment]
`ladspa` is now there. Trying to get `rubberband` to build here: #11570

**Comment by iMichka** [comment]
Fixed by #11570
